src/presentation/http/router/noteSettings.test.ts

```diff
@@ -395,6 +412,24 @@ describe('NoteSettings API', () => {
     });
 
     test('Returns status code 404 and "User does not belong to Note\'s team" message if no such a note exists', async () => {
+      // truncate tables which are needed
+      await global.db.query(`TRUNCATE public.notes, public.note_teams, public.users CASCADE`);
```

Comment:
What does this function do? Why is it here?

Reply:
it deletes all data which has been inserted in previous tests (it will be moved to beforeEach)